docs/website/02_examples/cantileverBeam.md

```diff
@@ -4,7 +4,143 @@ SPDX-License-Identifier: CC-BY-SA-4.0
+$\mathbf{B}$-operator and the local stiffness matrix are then obtained by
+```cpp
+Eigen::Matrix2Xd B;
+for (auto &gp : quadratureRule) { // (1)! 
+  basisW.evaluateJacobian(gp.position(), dNwDxi); // (2)! 
+  basisPhi.evaluateFunction(gp.position(), Nphi); // (3)! 
+  basisPhi.evaluateJacobian(gp.position(), dNphiDxi); // (4)! 
+
+  B.setZero(Eigen::NoChange, numDofsPerEle);
+
+  for (unsigned int i = 0; i < wFE.size(); ++i)
+    B(1, wFE.localIndex(i)) = dNwDxi[i] / detJ; // (5)! 
+
+  for (unsigned int i = 0; i < phiFE.size(); ++i)
+    B.col(phiFE.localIndex(i)) << dNphiDxi[i] / detJ, Nphi[i]; // (6)! 
+    
+  KLocal += B.transpose() * C * B * detJ * gp.weight(); // (7)! 
+}
+```
+
+1. Looping over the integration point
+2. derivative of the ansatz function of `w` with respect to the local parametric space
+3. ansatz function of `phi`
+4. derivative of the basis of `phi` with respect to the local parametric space
+5. Filling up the $\mathbf{B}$-operator for the columns corresponding to 'w'
+6. Filling up the $\mathbf{B}$-operator for the columns corresponding to 'phi'
+7. Integrating to arrive at the local stiffness matrix
+
+Assembly of the local element stiffness matrices is done to obtain the global element stiffness matrix as shown below:  
+```cpp
+for (auto i = 0U; i < localView.size(); ++i)
+  for (auto j = 0U; j < localView.size(); ++j)
+    KGlobal(localView.index(i)[0], localView.index(j)[0]) += KLocal(i, j);
+```
+The point load on the right end of the beam is applied by setting the corresponding entry in the global external load 
+vector to the prescribed value `F` using the command `#!cpp FExtGlobal(getGlobalDofId(TimoshenkoBeam::w, basis, L)) = F;`.
+The function 
+```cpp 
+unsigned int getGlobalDofId(TimoshenkoBeam requestedQuantity, const auto &basis, const double position);
+```
+with
+```cpp 
+enum class TimoshenkoBeam { w, phi };
+``` 
+is used to get the degree of freedom of the `requestedQuantity` (`w` or `phi`) for the defined `basis` at position `L`.
+
+The left end of the beam is clamped by using the following code:
+```cpp
+std::vector<unsigned int> fixedDofs{getGlobalDofId(TimoshenkoBeam::w, basis, 0.0),
+                                    getGlobalDofId(TimoshenkoBeam::phi, basis, 0.0)};
+for (auto dof : fixedDofs) {
+  KGlobal.col(dof).setZero();
+  KGlobal.row(dof).setZero();
+  KGlobal(dof, dof) = 1.0;
+}
+```
+Finally, the system of equations are solved by using the solver type `#!cpp Ikarus::SolverTypeTag::d_LDLT`. For more 
+details on the solver types, refer the [documentation](../01_framework/solvers.md).
+For post-processing, the deformed configuration of the beam can be plotted using the following function shown here in the example:
+```cpp
+void plotDeformedTimoschenkoBeam(auto &gridView, auto &basis, auto &d_glob, double EI, double GA, double L, double F);
+```
+This function uses the plotting features from [Matlab](https://de.mathworks.com/products/matlab.html).
```

Comment:
We should refer here to https://github.com/alandefreitas/matplotplusplus and say it has a similar syntax as matplotlib https://matplotlib.org/